api/base/settings/defaults.py

```diff
@@ -79,7 +79,7 @@
     # Needs to go before CommonMiddleware, so that transactions are always started,
     # even in the event of a redirect. CommonMiddleware may cause other middlewares'
     # process_request to be skipped, e.g. when a trailing slash is omitted
-    'api.base.middleware.DjangoGlobalMiddleware',
+    #'api.base.middleware.DjangoGlobalMiddleware',
```

Comment:
I was getting strange errors and it wasn't processing exceptions properly.  I noted in my pull request that I can comment it out and push again if you want.